app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -17,6 +17,19 @@ const Notes = () => {
   const [showDeleteAlert, setShowDeleteAlert] = useState(false);
   const [searchTerm, setSearchTerm] = useState("");
   const [showMenu, setShowMenu] = useState(false);
+  const [notesList, setNotes] = useState(notes);
```

Comment:
This is again a bad practice. State name and setter name should be consistent. It should always be:
```
state, setState
```

So here:
```suggestion
const [notes, setNotes] = useState(NOTES);
```